docker-compose.yaml

```diff
@@ -4,7 +4,6 @@ services:
       context: .
       target: production
     image: ffc-grants-eligibility-checker
-    # platform: ${PLATFORM}
```

Comment:
This is needed for ARM based architectures - why are we removing it here?

Reply:
It's been replaced.